portal/server/utils/destroy-session-and-redirect-to-start.js

```diff
@@ -1,17 +1,12 @@
-const { PORTAL_URL } = require('../constants');
-
 /**
  * Destroys a user's session and redirects them to the start page.
  * @param {Object} req
  * @param {Object} res
  * @returns {void}
  */
 function destroySessionAndRedirectToStart(req, res) {
-  const startPageRedirect = Boolean(process.env.START_PAGE_REDIRECT);
```

Comment:
Env variable START_PAGE_REDIRECT was not used in DEV, Staging or PROD. So removing this logic